airflow/example_dags/example_bash_operator.py

```diff
@@ -29,47 +29,43 @@
     'owner': 'airflow',
 }
 
-dag = DAG(
+with DAG(
     dag_id='example_bash_operator',
     default_args=args,
     schedule_interval='0 0 * * *',
     start_date=days_ago(2),
     dagrun_timeout=timedelta(minutes=60),
     tags=['example', 'example2'],
     params={"example_key": "example_value"},
-)
+) as dag:
 
-run_this_last = DummyOperator(
-    task_id='run_this_last',
-    dag=dag,
-)
+    run_this_last = DummyOperator(
+        task_id='run_this_last',
+    )
+
+    # [START howto_operator_bash]
+    run_this = BashOperator(
```

Comment:
Could you please update the documentation and remove the added indent? You can do it with " :dedent:" option. For example, see: https://github.com/apache/airflow/blob/master/docs/apache-airflow-providers-google/operators/cloud/bigquery.rst

Reply:
Thanks for the example, done! I went ahead and made this change in some other examples I hadn't modified as well (like in kubernetes.rst) that had similar indent issues.

While I was looking at this, I noticed that further down in the bash.rst (and probably other documentation files), the context manager is not used in code blocks that were written directly into the documentation, like in the last example [here](https://github.com/apache/airflow/blob/96150401c1344e818d91084da057824afd630781/docs/apache-airflow/howto/operator/bash.rst). I could update the documentation wherever this occurs if you thought that made sense (wasn't sure if it would be too far out of the scope of the original issue, though).